pandas/tests/io/formats/test_to_csv.py

```diff
@@ -6,6 +6,8 @@
 import numpy as np
 import pytest
 
+from pandas.compat import StringIO
```

Comment:
Can just import from io now

Reply:
Good to know it's Py3+ now. Ignoring as this dependency can actually be removed, thanks to your other comment pointing we don't need the `StringIO` at all :+1: